centaur/src/main/resources/standardTestCases/sizerelativepath/sizerelativepath.wdl

```diff
@@ -0,0 +1,32 @@
+version 1.0
+
+task print_size {
+    input {
+        File file
+    }
+    Int bytes = ceil(size(file))
+
+    command {
+        echo ~{bytes}
+    }
+
+    output {
+        String out = read_string(stdout())
+    }
+
+    runtime {docker: "ubuntu:latest"}
+}
+
+workflow sizerelativepath {
+    input {
+        File file = "LICENSE.txt"
```

Comment:
Instead of `LICENSE.txt` which might (??!) move or change from time to time, it might be better to reference  an explicitly deterministic file (eg `centaur/src/main/resources/known_size_file.txt`)